hub/util/tests/test_transform.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import hub
+from hub.util.transform import merge_tensor_metas
+from hub.core.meta.tensor_meta import TensorMeta
+from hub.core.tests.common import parametrize_all_dataset_storages
+
+
+@parametrize_all_dataset_storages
+def test_merge_tensor_metas(ds):
```

Comment:
Could you also add end to end transformation test to ensure that compressed data is generated properly using transforms?